frontend/app/src/screens/proposed-changes/proposed-change-details.tsx

```diff
@@ -23,23 +24,28 @@ import { useAtom } from "jotai";
 import { HTMLAttributes } from "react";
 import { useNavigate, useParams } from "react-router-dom";
 import { TaskDisplay } from "../branches/task-display";
+import LoadingScreen from "../loading-screen/loading-screen";
 import { getObjectPermissionsQuery } from "../permission/queries/getObjectPermissions";
 import { getPermission } from "../permission/utils";
-import {
-  BRANCH_MERGE_WORKFLOW,
-  BRANCH_REBASE_WORKFLOW,
-  BRANCH_VALIDATE_WORKFLOW,
-} from "../tasks/constants";
+import { PROPOSED_CHANGE_MERGE_WORKFLOW } from "../tasks/constants";
 
 export const ProposedChangeDetails = ({ className, ...props }: HTMLAttributes<HTMLDivElement>) => {
   const { proposedChangeId } = useParams();
   const [proposedChangesDetails] = useAtom(proposedChangedState);
 
   const navigate = useNavigate();
 
-  const { loading, data, error } = useQuery(
-    gql(getObjectPermissionsQuery(PROPOSED_CHANGES_OBJECT))
-  );
+  const { data } = useQuery(gql(getObjectPermissionsQuery(PROPOSED_CHANGES_OBJECT)), {
+    pollInterval: 2000,
+  });
+
+  const { loading: loadingCheck, data: checkData } = useQuery(TASK_DETAILS_CHECK, {
```

Comment:
check is already use in other domain of proposed change, my fear is that is can be confused with PC checks. 
 
 How about GET_TASK_COUNT? 
 checkData => mergeTaskData etc...